dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -7033,6 +7145,20 @@ definitions:
         format: "date-time"
         position: 6
         description: "Timestamp at which the notification was last updated"
+  Optional:
```

Comment:
This kind of bothers me.  `Try putting @ApiIgnore` or `ApiOperation(hidden = true)`, for the method and see if it gets rid of the this in swagger completely.